packages/daily-image-api/__tests__/integration/handlerGet.test.ts

```diff
@@ -113,3 +113,22 @@ describe("Test querying daily image", () => {
       .register(() => ({ dynamoDb }))
       .register(appDependencies);
 });
+
+describe("logEvent", () => {
+  test("Event is logged then next callback is called with event", async () => {
+    const originalConsoleLog = console.log;
+    console.log = jest.fn();
+
+    const expectedResolvedValue = "success";
+    const nextCallback = jest.fn().mockResolvedValue(expectedResolvedValue);
+    const expectedEvent = { test: "event" };
+    const expectedArgs = [1, 2, 3];
+
+    const result = await logEvent(nextCallback)(expectedEvent, ...expectedArgs);
```

Comment:
I've updated the master branch to allow the console to be passed in as a dependency (what you've called `expectedArgs`).

So you can pass the mock logger in this like now:
```
const dependencies: AppDependencies = { logger: mockLogger } as any; // was 'const expectedArgs = [1, 2, 3];'

const result = await logEvent(nextCallback)(expectedEvent, dependencies);
```